docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -9,6 +9,7 @@ New and Improved
 ----------------
 
 - Peaks can now be added or removed from a PeaksWorkspace using the :ref:`peaks overlay <sliceviewer_peaks_overlay>` in :ref:`sliceviewer`.
+- Improved cross-platform theming by forcing a material design icon for closing editor tabs.
```

Comment:
So this release note needs a tweak, it's telling users what is happening instead of what it means for them. Could we change it to something like:

`Improved cross-platform theming by using the same icon on all platforms for closing editor tabs`

If they care about material design they'll instantly recognise it as such, but I suspect most people don't care :wink: